src/DocumentedDocument.spec.ts

```diff
@@ -0,0 +1,140 @@
+import * as Document from "./Document";
+import * as DocumentedDocument from "./DocumentedDocument";
+import DefaultExport from "./DocumentedDocument";
+import * as Documents from "./Documents";
+import { clazz, extendsClass, hasDefaultExport, hasMethod, hasProperty, interfaze, module, OBLIGATORY, STATIC } from "./test/JasmineExtender";
+
+describe( module( "Carbon/DocumentedDocument" ), ():void => {
```

Comment:
`DocumentedDocument` may be interpreted as a document that was documented in the documentation... (or something like that). Instead please rename this class to `ServiceAwareDocument`

Reply:
Perfect, I could'd find a suitable name for this interface.